packages/bson-bench/src/base.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+const {
+  Package,
+  BSONLib,
+  BenchmarkSpecification,
+  RunBenchmarkMessage,
+  BenchmarkResult,
+} = require(`${__dirname}/common`);
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+const { performance } = require("perf_hooks");
```

Comment:
Any reason to not use `.mjs` and es module syntax here?

Reply:
No, just didn't occur to me. Fixing now.